src/tutorials/data/queries.md

```diff
@@ -0,0 +1,365 @@
+
+
+## Sort in CouchDB
+
+When dealing with various types of data, the ascending sort order is the following:
+
+- null
+- booleans
+- numbers
+- strings: “a” < “A” < ”aa” < “b” …
+- arrays
+- objects
+
+See the [CouchDB documentation](https://docs.couchdb.org/en/2.3.1/ddocs/views/collation.html#collation-specification) for more details on the sort orders.
+
+ℹ️ The sort order is the same for Mango queries and views. 
+
+### Comparison of strings
+
+Comparison of strings is done using ICU which implements the Unicode Collation Algorithm, giving a dictionary sorting of keys. This can give surprising results if you were expecting ASCII ordering. Note that:
+
+- All symbols sort before numbers and letters (even the “high” symbols like tilde, `0x7e`)
+- Differing sequences of letters are compared without regard to case, so `a < aa` but also `A < aa` and `a < AA`
+- Identical sequences of letters are compared with regard to case, with lowercase before uppercase, so `a < A`.
+
+
+## Views
+
+⚠️ In Cozy, we disabled by default the possibility to create views from applications. Consequently, [cozy-client](https://github.com/cozy/cozy-client) does not support views creation, because they are disabled for applications. See the [view performances](./advanced.md#views-performances) section to find out why. 
+
+[CouchDB views](https://docs.couchdb.org/en/2.3.1/ddocs/views/index.html) are another way to express queries in Cozy, used by the [cozy-stack](https://github.com/cozy/cozy-stack/blob/331f6f39402dc7ec38ecf2f5ff408a1b5cf3d730/pkg/couchdb/index.go#L48-L229). It consists of creating [map-reduce](https://en.wikipedia.org/wiki/MapReduce) functions, where the `map` is used to qualify which documents should be indexed, and the optional `reduce`  can be used to compute aggregate functions on them.
+
+Views are much more flexible than Mango queries, as the `map`  is a user-defined function (UDF) that can be written in JavaScript.
+
+For example:
+
+```javascript
+function(doc) {
+  if(doc.category && doc.created_at) {
+    emit(doc.category, doc.created_at);
+  }
+}
+```
+
+
+A `map` function must always produce an `emit` , with the key and the value respectively in first and second argument.
+
+ℹ️ Behind the hoods, views are represented exactly like the Mango queries: both uses [B+ Tree](./advanced.md#indexes-concepts), with values stored in the leafs.
+
+### Query a view
+
+Just like the Mango queries, views are particularly suited for equality and range queries. See the [CouchDB documentation](https://docs.couchdb.org/en/2.3.1/api/ddoc/views.html) to know more about the API.
+
+To find a specific document, use the  `key` parameter:
+
+`key="sport"`
+
+To find a range, combine the `startkey` and `endkey` parameters:
+
+`startkey="category1"&endkey="category9"`
```

Comment:
I'm not sure it is a very good example, as `category42` is between `category1` and `category9`. Maybe add padding: `category0001` and `category0009`.